src/main/java/pl/project13/maven/git/GitCommitIdMojo.java

```diff
@@ -472,6 +476,16 @@ void loadBuildTimeData(@NotNull Properties properties) {
     put(properties, BUILD_TIME, smf.format(buildDate));
   }
 
+  void loadBuildHostData(@NotNull Properties properties) {
+    String buildHost = null;
+    try {
+      buildHost = InetAddress.getLocalHost().getHostName();
+    } catch (UnknownHostException e) {
+      log("Unable to get build host");
```

Comment:
I have updated the error message to be more verbose.

> And indeed don't set it if it's null.

We have to put null value explicitly, because null is processed into "Unknown" later. I think it would be better to have "Unknown" value in the resulting property rather than omitting the property completely from the list of properties. Someone may get confused in case if property does not exist for some non-obvious reason.